app/views/posts/show.html.erb

```diff
@@ -1 +1,31 @@
-<h1 class="post_title">post:</h1>
+<%= link_to user_posts_path(@user.id) do %>
+  < Back to <%= @user.name %> posts
+<% end %>
+
+<div>
+  <div>
+    <h2>Post <%= @post.id %> by <%= @user.name %></h2>
+    <ul>
+      <li>
+        Comments: <%= @post.comments.count %>
+      </li>
+      <li>
+        Likes: <%= @post.likes.count %>
+      </li>
+    </ul>
+  </div>
+
```

Comment:
- [ ] One of the requirements for this project is to ensure that different views are identical to that of the wireframe provided for the view. On the `Posts#Show` page, the details about **_the comments and likes for the post_** should be on the same row with the **_user's name and the post's title_** but at the far end. 👍 🖱️ 

### Your `Posts#Show` Page

![image](https://github.com/Felipe-Perez-Ferraro/blog_app/assets/77106962/cb8a2059-fc4a-4df5-a064-d0cd951f2a66)

### Wireframe Provided

![image](https://github.com/Felipe-Perez-Ferraro/blog_app/assets/77106962/017b23f7-01cf-40f5-8601-23c35a6712ab)